projects/typescript-vue/src/app.vue

```diff
@@ -0,0 +1,212 @@
+            <span class="title">bpmn-visualization + Vue</span>
+        </section>
+        <section>
+            <div>
+                <input type="file" id="btn-open-file" @change="changeDiagramm" name="btn-open-file" class="hidden" />
+                <label for="btn-open-file" class="btn btn-primary has-icon-right"><span>Choose a local file </span><span
+                        class="icon icon-upload"></span></label>
+            </div>
+            <div class="dropdown mr-2" :class="{ active: dropdownActive }">
+                <a href="#" class="btn" @click="dropdownActive = !dropdownActive" tabindex="0">
+                    Edit Nodes <i class="icon icon-caret"></i>
+                </a>
+                <!-- menu component -->
+                <ul class="menu" style="color: #2c3e50; width: 400px;">
+                    <div class="form-group">
+                        <label class="form-label label-sm">Node</label>
+                        <select class="form-select select-sm" v-model="nodeDropdown"
+                            @change="setSelectedElement(nodeDropdown)">
+                            <option v-for="opt in allFlowNodes" :value="opt.bpmnSemantic.id">{{ opt.bpmnSemantic.name ||
+                                opt.bpmnSemantic.id }} ({{ opt.bpmnSemantic.kind }})</option>
+                        </select>
+                    </div>
+                    <div class="form-group">
+                        <label class="form-label label-sm">Overlays</label>
+                        <div style="display: flex; gap: 10px;">
+                            <input class="form-input input-sm" v-model="overlayText" />
+                            <select class="form-select select-sm" v-model="overlayPosition"
+                                @change="setSelectedElement(nodeDropdown)">
+                                <option
+                                    v-for="pos in ['top-left', 'top-right', 'top-center', 'bottom-left', 'bottom-right', 'bottom-center', 'middle-left', 'middle-right']"
+                                    :value="pos">{{ pos }}</option>
+                            </select>
+                            <button class="btn btn-sm" @click="addOverlay">Add Overlay</button>
+                        </div>
+                        <div style="display: flex; place-content: end; align-items: center; height: 40px; ">
+                            <button class="btn btn-error btn-sm" style="float: right;" @click="removeAllOverlays">Remove all
+                                Overlays</button>
+                        </div>
+                    </div>
+                </ul>
+            </div>
+        </section>
+        <section style="flex-grow: 0">
+            <div class="dropdown">
+                <a href="#" class="btn dropdown-toggle" tabindex="0">
+                    Docs <span class="icon icon-caret icon-in-the-middle"></span>
+                </a>
+                <ul class="menu">
```

Comment:
**suggestion:** I am not sure we need to provide the exact same header than in the regular examples here. We don't do it of other projects. Let me think about this.
Nice to have anticipated #269